tailwind.config.ts

```diff
@@ -41,6 +41,16 @@ const config: Config = {
           200: '#482820',
         },
       },
+      opacity: {
```

Comment:
bit confused why you need this. i will remove this since tailwind already provides opacity